src/ajax/xhr.js

```diff
@@ -73,12 +77,14 @@ jQuery.ajaxTransport( function( options ) {
 
 								// File: protocol always yields status 0; see #8605, #14207
 								xhr.status,
-								xhr.statusText
+								xhr.statusText,
+								""
 							);
 						} else {
 							complete(
 								xhrSuccessStatus[ xhr.status ] || xhr.status,
 								xhr.statusText,
+								support.responseURL ? xhr.responseURL : "",
```

Comment:
What would happen if we just passed it through without checking for support? If IE 11 doesn't crash but just passes `undefined` through, that'd be fine for me.

Reply:
removed this condition as well, IE doesn't crash and passes undefined